src/plugins/template/src/compiled_model.cpp

```diff
@@ -124,17 +128,8 @@ void TemplatePlugin::CompiledModel::compile_model(const std::shared_ptr<ov::Mode
 // ! [executable_network:map_graph]
 
 // ! [executable_network:create_infer_request]
-std::shared_ptr<InferenceEngine::IInferRequestInternal> TemplatePlugin::CompiledModel::create_infer_request() const {
-    auto internal_request = create_sync_infer_request();
-    return std::make_shared<TemplateAsyncInferRequest>(
-        std::static_pointer_cast<TemplatePlugin::TemplateInferRequest>(internal_request),
-        get_task_executor(),
-        get_template_plugin()->_waitExecutor,
-        get_callback_executor());
-}
-
-std::shared_ptr<InferenceEngine::IInferRequestInternal> TemplatePlugin::CompiledModel::create_sync_infer_request()
-    const {
+std::shared_ptr<ov::IAsyncInferRequest> TemplatePlugin::CompiledModel::create_infer_request() const {
+    // auto internal_request = create_sync_infer_request();
```

Comment:
remove commented code?

Reply:
This code will used after the migration to new API